modules/core/pom.xml

```diff
@@ -49,6 +49,14 @@
       <groupId>org.apache.accumulo</groupId>
       <artifactId>accumulo-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
It would be nice if fluo core did not depend on these, and only fluo command depended on them.  Maybe fluo command code could optional pass a `Function<RowColumnValue, String>` to fluo core code.  This function could call commons-csv code and be defined in fluo command.  This function may be a horrible idea I have not given it a lot of thought.